### ISSUE TITLE
Overall stats for gedcom diff

### ISSUE THREAD

**Issue by nev7n** [elliotchance/gedcom#200]
A nice feature would be to give overall stats in gedcom diff.

For example:
- number of people each gedcom
- number identical matches
- number partial matches
- number unique in left
- number unique in right
- average similarity

**Comment by elliotchance** [comment]
Yep, that's a good idea. Especially when transferring details from different trees it can be used as a progress indicator. And who doesn't love stats!? 😄

**Comment by elliotchance** [comment]
That's great to hear because thats the exact use case I built it for. I hope you don't mind, I've added you to the gedcom-app gitter since it's a better place to chat than issues.

**Comment by elliotchance** [comment]
Sorry, I couldn't add you without an email: https://gitter.im/gedcom-app/community
